src/forms/Field.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { ChangeEvent, useMemo } from "react"
 import { ErrorMessage } from "../notifications/ErrorMessage"
 import { UseFormMethods, RegisterOptions } from "react-hook-form"
+import { isURL } from "class-validator"
```

Comment:
@ludtkemorgan just wondering if you have any concerns about adding class-validator to uic

Reply:
edited existing urlRegex. As far as i know we were using it only for Open house link, but it also had that bug when adding link without protocol.